test/index.js

```diff
@@ -29,7 +29,21 @@ const RABBIT_MQ_URI = process.env.RABBIT_MQ_URI || 'amqp://localhost';
 
 describe('socket.io-amqp', function ()
 {
-    this.timeout(10000);
+    this.timeout(1000);
+
+    const sios = [];
+
+    after(function ()
```

Comment:
would it make more sense to make this an afterEach so we're cleaning up between tests?

Reply:
It leads to some concurrency issues within close function of the socket.io as discussed.